custom_components/tahoma/number.py

```diff
@@ -28,6 +28,36 @@
         max_value=4,
         entity_category=EntityCategory.CONFIG,
     ),
+    OverkizNumberDescription(
+        key=OverkizState.CORE_ECO_ROOM_TEMPERATURE,
+        name="Eco temperature",
+        icon="mdi:thermometer",
+        command=OverkizCommand.SET_ECO_TEMPERATURE,
+        min_value=6,
+        max_value=29,
+        step=0.5,
+        entity_category=EntityCategory.CONFIG,
+    ),
+    OverkizNumberDescription(
+        key=OverkizState.CORE_COMFORT_ROOM_TEMPERATURE,
+        name="Comfort temperature",
+        icon="mdi:home-thermometer-outline",
+        command=OverkizCommand.SET_COMFORT_TEMPERATURE,
+        min_value=7,
+        max_value=30,
+        step=0.5,
+        entity_category=EntityCategory.CONFIG,
+    ),
+    OverkizNumberDescription(
+        key=OverkizState.CORE_SECURED_POSITION_TEMPERATURE,
+        name="Freeze protection temperature",
+        icon="mdi:sun-thermometer-outline",
+        command=OverkizCommand.SET_SECURED_POSITION_TEMPERATURE,
+        min_value=5,
+        max_value=15,
+        step=0.5,
+        entity_category=EntityCategory.CONFIG,
+    ),
```

Comment:
On your device, you can set the 3 temperatures at the same time? The 3 commands are always available or only one?

Reply:
Could it be because this is set to config? So it will only be visible on the device page via Integrations -> Devices.

Reply:
This is all I have for the device: 
![image](https://user-images.githubusercontent.com/186777/149379265-c19082a8-cbca-4b5c-b155-c582f183a614.png)
We can see the mode change, but not the 3 others number settings.

Reply:
Do you have any errors in your log and which version of HA do you run? I think I have some feeling what is wrong. If you apply this fix -> https://github.com/iMicknl/ha-tahoma/pull/667, your dev works again.

Reply:
I will try to apply the fix. I'm running: Home Assistant 2021.12.9